02- Running ADO Scanner from command line/Readme.md

```diff
@@ -131,3 +131,24 @@ Some AzSK.ADO controls require graph access for correct evaluation. To fetch the
 ```PowerShell
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -UseGraphAccess
 ```
+
+----------------------------------------------
+
+### Scan ALT-account related controls using "-ALTControlEvaluationMethod" parameter
```

Comment:
add a pointer for this portion and "-UseGraphAccess"  portion in the beginning of this page i.e. at "Contents"

Reply:
done